test/amoc_api_helper.erl

```diff
@@ -0,0 +1,54 @@
+-module(amoc_api_helper).
```

Comment:
Beautiful module :) However, one thing is missing. In line 39 you start fusco which gives you a client, for each request. [fusco:start/2](https://github.com/esl/fusco/blob/master/src/fusco.erl#L77) create a new gen_server process and it is not killed after the request is done - which is bad because if someone make use of the utils module and his or her test SUITE will generate 10000000000 request the he or she will end up with many orphaned processes. There seem to be 2 solutions:

* either add an API for obtaining/releasing (get_client/0, release_client/1) the HTTP client and then pass it on the the requests, or
* disconnect the client after each request.

Think which one is better and choose @kklocek !
*